src/main/java/com/fasttime/domain/resume/exception/NoResumeWriterException.java

```diff
@@ -0,0 +1,20 @@
+package com.fasttime.domain.resume.exception;
+
+import com.fasttime.global.exception.ApplicationException;
+import com.fasttime.global.exception.ErrorCode;
+
+public class NoResumeWriterException extends ApplicationException {
+    private static final ErrorCode ERROR_CODE = ErrorCode.HAS_NO_PERMISSION_WITH_THIS_RESUME;
```

Comment:
구글 스타일에 맞게 공백 추가해주세요~
```suggestion
public class NoResumeWriterException extends ApplicationException {

    private static final ErrorCode ERROR_CODE = ErrorCode.HAS_NO_PERMISSION_WITH_THIS_RESUME;
```